data_summaries/Cidar.json

```diff
@@ -0,0 +1,32 @@
+{
+  "dataset-name (must match unique identifier)": {
+    "Unique Dataset Identifier": "cidar",
+    "Dataset Name": "cidar",
+    "Paper Title": "CIDAR: Culturally Relevant Instruction Dataset For Arabic",
+    "Dataset URL": "https://github.com/ARBML/CIDAR",
+    "GitHub URL": "https://github.com/ARBML/CIDAR",
+    "Hugging Face URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Papers with Code URL": "https://paperswithcode.com/dataset/cidar",
+    "ArXiv URL": "https://arxiv.org/abs/2402.03177",
+    "Semantic Scholar Corpus ID": "",
+    "Collection": "Cidar",
+    "Collection URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Languages": ["Arabic"],
+    "Task Categories": ["Instruction Generation"],
+    "Text Sources": ["huggingface.co/datasets/mlabonne/alpagasus"],
```

Comment:
Text Sources should actually be: https://learning.aljazeera.net/ar/asktheteacher.

Reading further, the other examples are from Stanford's 52k Alpaca, which were generated directly by GPT-3 (add to model generated list), and then they filtered using GPT-4.